docs/protocol/ISM/multisig-ISM.mdx

```diff
@@ -1,44 +1,63 @@
 # Multisig ISM
 
+_Verify messages using validator signatures_
+
 The `MultisigISM` is one of the most commonly used ISM types. These ISMs verify that `m` of `n` [Validators](../agents/validators.mdx) have attested to the validity of a particular interchain message.
 
 ## Multisig Module Types
 
 Based on the following module types, the relayer will encode the appropriately formatted metadata:
 
 ```solidity file=<rootDir>/node_modules/@hyperlane-xyz/core/contracts/interfaces/IInterchainSecurityModule.sol#L5-L14
+
 ```
 
-For the aforementioned module types, the following metadata formats are satisfied by the relayer for use by the ISMs:
+For these module types, the metadata is formatted by the relayer as follows:
```

Comment:
this whole bit is a bit confusing, think we should have a handcraft enum that just focuses on the two types we care about e.g.

```
## Multisig Module Types

There are two types of Multisig ISMs modules:

` ` `solidity
  enum Types {
      ...
      MERKLE_ROOT_MULTISIG,
      MESSAGE_ID_MULTISIG,
      ...
  }
` ` `
```

and remove "For these module types, the metadata is formatted by the relayer as follows:"